mappings/net/minecraft/util/registry/RegistryTracker.mapping

```diff
@@ -0,0 +1,47 @@
+CLASS net/minecraft/class_5455 net/minecraft/util/registry/RegistryTracker
+	FIELD field_25918 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_25919 INFOS Ljava/util/Map;
+	METHOD method_30518 getDimensionTypeRegistry ()Lnet/minecraft/class_2378;
+	METHOD method_30519 load (Lnet/minecraft/class_3300;)Lnet/minecraft/class_5455$class_5457;
+		ARG 0 resourceManager
+	METHOD method_30520 register (Lcom/google/common/collect/ImmutableMap$Builder;Lnet/minecraft/class_5321;Lcom/mojang/serialization/MapCodec;Z)Lcom/google/common/collect/ImmutableMap$Builder;
+		ARG 0 infosBuilder
+		ARG 1 registryRef
+		ARG 2 elementCodec
+		ARG 3 synced
+	METHOD method_30524 addBuiltinEntries (Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_2378;)V
+		ARG 0 tracker
+		ARG 1 registry
+	METHOD method_30525 setupBuiltin (Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_5321;)V
+		ARG 0 tracker
+		ARG 1 registryRef
+	METHOD method_30526 load (Lnet/minecraft/class_5382;Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_5455$class_5456;)V
+		ARG 0 ops
+		ARG 1 tracker
+	METHOD method_30527 getRegistrySafely (Lnet/minecraft/class_5321;)Ljava/util/Optional;
+		ARG 1 registryRef
+	METHOD method_30528 create ()Lnet/minecraft/class_5455$class_5457;
+	METHOD method_30530 getRegistry (Lnet/minecraft/class_5321;)Lnet/minecraft/class_2385;
+		ARG 1 registryRef
+	CLASS class_5456 Info
```

Comment:
Maybe something like `Config`? (assuming that all the fields are final here, if not maybe `State`?)

Reply:
this is definitely not a state. this is just a pojo dictating handling of registry elements in the tracker

Reply:
Yea, that's why I first proposed `Config`

Reply:
Not a config either as it is not configurable at all